package.json

```diff
@@ -51,6 +51,7 @@
     "nodemailer": "^6.9.14",
     "passport-jwt": "^4.0.1",
     "pg": "^8.12.0",
+    "postgres": "^3.4.4",
```

Comment:
what's the essence of the package?

Reply:
i initially had issues running the repo. This was one of the fixes i tried. I'd take it off now